test/shared/src/test/scala/zio/test/TestMain.scala

```diff
@@ -1,12 +1,12 @@
 package zio.test
-
+/*
 import scala.concurrent.ExecutionContext.Implicits.global
 
 import zio.test.mock._
 import zio.test.TestUtils.{ report, scope }
-
+ */
 object TestMain {
-
+  /*
```

Comment:
Accidentally commented?

Reply:
Temporarily. I'm testing `MockSpec` using ZIO Test (see `MockSpecSpec.scala`) and during development I have `~testJVM/test:run` running, but sbt could see two main classes and kept asking to choose one. Will restore this after I'm done.